### ISSUE TITLE
Thread 'tokio-runtime-worker-6' panicked at 'specified instant was later than self', src/libstd/sys/unix/time.rs:298

### ISSUE THREAD

**Issue by LaggAt** [paritytech/parity-ethereum#10478]
- **Parity Ethereum version**: 2.3.5 - Parity-Ethereum/v2.3.5-stable-ebd0fd0-20190227/x86_64-linux-gnu/rustc1.32.0
- **Operating system**: Ubuntu 16.04.5 LTS
- **Installation**: Docker container parity/parity:stable
- **Fully synchronized**: no
- **Network**: ethereum
- **Restarted**: yes

_Your issue description goes here below. 

**actual**
Parity throws an exception every few minutes. Sometimes it gets stuck and continuously crasches after some minutes, but it continues to catch up later. e.g. For block 2396834 it took estimated 5 Hours before it continued to work.

Here is a log where parity crashed, and as a result the docker container is restarting:

```
14.3.2019 20:05:032019-03-14 19:05:03 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #30796   24/25 peers     10 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:092019-03-14 19:05:09 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #34740   24/25 peers     10 MiB chain  0 bytes db   58 KiB queue   56 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:142019-03-14 19:05:14 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #39702   24/25 peers      9 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:192019-03-14 19:05:19 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #42608   25/25 peers     10 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:242019-03-14 19:05:24 UTC Syncing #2396834 0xe485…aa2a     0.20 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #42608   25/25 peers     40 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:322019-03-14 19:05:31 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #42608   25/25 peers     38 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:362019-03-14 19:05:36 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #42608   19/25 peers     19 MiB chain  0 bytes db   58 KiB queue   54 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:412019-03-14 19:05:41 UTC Syncing #2396834 0xe485…aa2a     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+   43 Qed    #27236   15/25 peers     10 MiB chain  0 bytes db   58 KiB queue   53 MiB sync  RPC:  0 conn,    0 req/s, 3083 µs
14.3.2019 20:05:44
14.3.2019 20:05:44
14.3.2019 20:05:44====================
14.3.2019 20:05:44
14.3.2019 20:05:44stack backtrace:
14.3.2019 20:05:44   0:     0x55f51e238b3d - <no info>
14.3.2019 20:05:44   1:     0x55f51e237c62 - <no info>
14.3.2019 20:05:44   2:     0x55f51e236e16 - <no info>
14.3.2019 20:05:44   3:     0x55f51ca07228 - <no info>
14.3.2019 20:05:44   4:     0x55f51e2a9319 - <no info>
14.3.2019 20:05:44   5:     0x55f51e2a9004 - <no info>
14.3.2019 20:05:44   6:     0x55f51e2b4288 - <no info>
14.3.2019 20:05:44   7:     0x55f51e2a34c3 - <no info>
14.3.2019 20:05:44   8:     0x55f51e10bd66 - <no info>
14.3.2019 20:05:44   9:     0x55f51e10a036 - <no info>
14.3.2019 20:05:44  10:     0x55f51c8e8a55 - <no info>
14.3.2019 20:05:44  11:     0x55f51e0e73c1 - <no info>
14.3.2019 20:05:44  12:     0x55f51e0ea691 - <no info>
14.3.2019 20:05:44  13:     0x55f51e2bfdc9 - <no info>
14.3.2019 20:05:44  14:     0x55f51e0e8a3b - <no info>
14.3.2019 20:05:44  15:     0x55f51e0e1f95 - <no info>
14.3.2019 20:05:44  16:     0x55f51e0e102d - <no info>
14.3.2019 20:05:44  17:     0x55f51e0c3186 - <no info>
14.3.2019 20:05:44  18:     0x55f51e0c3327 - <no info>
14.3.2019 20:05:44  19:     0x55f51e0c2fd1 - <no info>
14.3.2019 20:05:44  20:     0x55f51e0babb9 - <no info>
14.3.2019 20:05:44  21:     0x55f51e0e94f2 - <no info>
14.3.2019 20:05:44  22:     0x55f51e0e8dc9 - <no info>
14.3.2019 20:05:44  23:     0x55f51e0eac8d - <no info>
14.3.2019 20:05:44  24:     0x55f51e0ea6db - <no info>
14.3.2019 20:05:44  25:     0x55f51e2bfdc9 - <no info>
14.3.2019 20:05:44  26:     0x55f51e0e556f - <no info>
14.3.2019 20:05:44  27:     0x55f51e2b395d - <no info>
14.3.2019 20:05:44  28:     0x7fd28ef306b9 - <no info>
14.3.2019 20:05:44  29:     0x7fd28ea5041c - <no info>
14.3.2019 20:05:44  30:                0x0 - <no info>
14.3.2019 20:05:44
14.3.2019 20:05:44Thread 'tokio-runtime-worker-6' panicked at 'specified instant was later than self', src/libstd/sys/unix/time.rs:298
14.3.2019 20:05:44
14.3.2019 20:05:44This is a bug. Please report it at:
14.3.2019 20:05:44
14.3.2019 20:05:44    https://github.com/paritytech/parity-ethereum/issues/new
14.3.2019 20:05:44
14.3.2019 20:05:442019-03-14 19:05:44 UTC Finishing work, please wait...
14.3.2019 20:07:22
14.3.2019 20:07:22
14.3.2019 20:07:22====================
14.3.2019 20:07:22
14.3.2019 20:07:22stack backtrace:
14.3.2019 20:07:22   0:     0x55f51e238b3d - <no info>
14.3.2019 20:07:22   1:     0x55f51e237c62 - <no info>
14.3.2019 20:07:22   2:     0x55f51e236e16 - <no info>
14.3.2019 20:07:22   3:     0x55f51ca07228 - <no info>
14.3.2019 20:07:22   4:     0x55f51e2a9319 - <no info>
14.3.2019 20:07:22   5:     0x55f51e2a9004 - <no info>
14.3.2019 20:07:22   6:     0x55f51e2b4288 - <no info>
14.3.2019 20:07:22   7:     0x55f51e2a34c3 - <no info>
14.3.2019 20:07:22   8:     0x55f51e10bd66 - <no info>
14.3.2019 20:07:22   9:     0x55f51e10a036 - <no info>
14.3.2019 20:07:22  10:     0x55f51c8e8a55 - <no info>
14.3.2019 20:07:22  11:     0x55f51e0e73c1 - <no info>
14.3.2019 20:07:22  12:     0x55f51e0ea691 - <no info>
14.3.2019 20:07:22  13:     0x55f51e2bfdc9 - <no info>
14.3.2019 20:07:22  14:     0x55f51e0e8a3b - <no info>
14.3.2019 20:07:22  15:     0x55f51e0e1f95 - <no info>
14.3.2019 20:07:22  16:     0x55f51e0e102d - <no info>
14.3.2019 20:07:22  17:     0x55f51e0c3186 - <no info>
14.3.2019 20:07:22  18:     0x55f51e0c3327 - <no info>
14.3.2019 20:07:22  19:     0x55f51e0c2fd1 - <no info>
14.3.2019 20:07:22  20:     0x55f51e0babb9 - <no info>
14.3.2019 20:07:22  21:     0x55f51e0e94f2 - <no info>
14.3.2019 20:07:22  22:     0x55f51e0e8dc9 - <no info>
14.3.2019 20:07:22  23:     0x55f51e0eac8d - <no info>
14.3.2019 20:07:22  24:     0x55f51e0ea6db - <no info>
14.3.2019 20:07:22  25:     0x55f51e2bfdc9 - <no info>
14.3.2019 20:07:22  26:     0x55f51e0e556f - <no info>
14.3.2019 20:07:22  27:     0x55f51e2b395d - <no info>
14.3.2019 20:07:22  28:     0x7fd28ef306b9 - <no info>
14.3.2019 20:07:22  29:     0x7fd28ea5041c - <no info>
14.3.2019 20:07:22  30:                0x0 - <no info>
14.3.2019 20:07:22
14.3.2019 20:07:22Thread 'tokio-runtime-worker-9' panicked at 'specified instant was later than self', src/libstd/sys/unix/time.rs:298
14.3.2019 20:07:22
14.3.2019 20:07:22This is a bug. Please report it at:
14.3.2019 20:07:22
14.3.2019 20:07:22    https://github.com/paritytech/parity-ethereum/issues/new
14.3.2019 20:07:22
14.3.2019 20:13:562019-03-14 19:13:56 UTC Shutdown is taking longer than expected.
14.3.2019 20:17:552019-03-14 19:17:55 UTC Shutdown timeout reached, exiting uncleanly.
```

config.toml:
```
[parity]
mode = "active"
auto_update = "all"
release_track = "stable"

[rpc]
interface = "all"

[websockets]
interface = "all"
origins = ["parity://*", "chrome-extension://*", "moz-extension://*", ""]

[stratum]
interface = "all"

[footprint]
pruning = "archive"
scale_verifiers = true
num_verifiers = 8

[secretstore]
http_interface = "all"

[network]
min_peers = 25
max_peers = 75

[misc]
color = false
```

**expected behavoir**

Parity should sync without crashing.

**steps to reproduce**

I just run the docker-container. Nothing special.

**Comment by LaggAt** [comment]
fyi: i gave the docker image more RAM, and it's not crashing anymore.

**Comment by ascjones** [comment]
Could be a duplicate of https://github.com/paritytech/parity-ethereum/issues/10419 (fixed in https://github.com/paritytech/parity-ethereum/pull/10423)

**Comment by joshua-mir** [comment]
@ascjones that fix should be present in 2.3.5

**Comment by joshua-mir** [comment]
@LaggAt if you see this again, could you please comment on #10419 and we'll reopen that issue? Closing this as you can't replicate anymore.
